.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
-benchmark/node_modules
```

Comment:
the above `node_modules` will ignore `node_modules` recursively throughout the repo